bazeldnf/extensions.bzl

```diff
@@ -202,6 +203,12 @@ def _handle_lock_file(config, module_ctx, registered_rpms = {}):
     content = module_ctx.read(config.lock_file)
     lock_file_json = json.decode(content)
 
+    if not config.ignore_deps:
+        if versions.is_at_least("7", versions.get()) and not versions.is_at_least("7.4.0", versions.get()):
```

Comment:
where are these constraints coming from?

Reply:
I don't remember exactly why, what I do remember is setting the bazel version to 7.0 and so until 7.4.0 with bazelisk and it was failing with some weird error, same for 8.x